applications/plugins/SofaPython/Binding_Node.cpp

```diff
@@ -92,7 +92,7 @@ static PyObject * Node_reset(PyObject * self, PyObject * /*args*/) {
 static PyObject * Node_init(PyObject * self, PyObject * /*args*/) {
     Node* node = get_node(self);
 
-    getSimulation()->init(node);
+    node->init(ExecParams::defaultInstance());
```

Comment:
I don't remember what does getSimulation() return?
does this change the behavior?

Reply:
Yes it does change the behavior...
But given how bad is the getSimulation() used in this context this is a good news :) 
Actually, getSimulation() returns the Simulation singleton (which shouldn't exists ;)). Then on this singleton the init method is there to initialize only the root of scene, this is why among other thing it add AnimationLoop/VisualLoop is there is none in the node. 
But...the init in binding_node is supposed to just bind the init method of a given Node.

If the Simulation::init() method is needed in python then it must be binding properly with a as a static method (or a function): 
Eg: 
```cpp
Sofa.Simulation.init(aRootNode)
```